packages/bech32/src/decode.ts

```diff
@@ -0,0 +1,12 @@
+import { BECH32_ADDRESS_LENGTH } from './penumbra-bech32';
+import { bech32m } from 'bech32';
+
+export const bech32ToAddress143 = (addr: string): { inner: Uint8Array } => {
+  const decodeAddress = bech32m.decode(addr, BECH32_ADDRESS_LENGTH);
+  return { inner: new Uint8Array(bech32m.fromWords(decodeAddress.words)) };
+};
+
+export const bech32ToDefault90 = (addr: string): Uint8Array => {
+  const decodeAddress = bech32m.decode(addr);
+  return new Uint8Array(bech32m.fromWords(decodeAddress.words));
+};
```

Comment:
this was formerly called "bech32ToUint8Array" though its behavior was subtly different than one might expect, given that name.